include/picongpu/particles/ParticlesInit.kernel

```diff
@@ -42,7 +42,7 @@ namespace picongpu
      *
      *  That means:
      *     The REAL number of particles density in units of volume**-3,
-     *       normed to UNIT_LENGHT**3
```

Comment:
please move changes from this file into a separate PR, even if it is doubling the work. The reason is that bose changes are independent from each other.